### PR TITLE
Adds plastic to cargo crates and adds in crates with 5 stacks of metal/glass

### PR DESCRIPTION
**Pull request by jakmak5**
To the cargo supplies section this adds 3 new crates. one is 10 sheets of plastic for 25$. The other two are glass in metal crates each one with 5 stacks of metal/glass for the price of 40$. I am willing to raise the price on either crate.

### CODE REVIEW

**Comment by PJB3005** [comment]
There's already material  sheet crates literally why.

**Comment by SonixApache** [comment]
Plastic seems alright. Literally why on the other crates.

**Comment by jakmak5** [comment]
again this one is to remove the tedium of having to order 8 crates of metal and glass for your autism fort.
Besides it doesnt make much sense for NT to waste all those crates on 12 orders of metal when they all could be put in a single crate.

**Comment by PJB3005** [comment]
You love removing "tedium" don't you? 

Having them be a ton of crates is GOOD, finally fucking something for cargo to do.

**Comment by Kurfursten** [comment]
I'm not sure about the plastic crate. If plastic is just too hard to get from botany, why don't we change the plastic sheet reaction? Here are some possibilities.
* Change the recipe to require only sulfuric acid. This makes it much less hassle to transform sheets over making polyacid.
* Get rid of the recipe entirely. Add a reaction_turf for plasticide that causes it to harden into sheets instantly when poured out.

Either would lower the barrier for plastic considerably.

**Comment by SonixApache** [comment]
plasticide + welding fuel

**Comment by Kurfursten** [comment]
You could even just use universal enzyme as a catalyst. It's next door in the kitchen.

**Comment by Zth--** [comment]
No. I was a botanist player for a long time and I can tell you that plastic is very important for the mechanic. I used to make tons of plastic for them and in exchange they gave me a chemistry dispenser. With this PR you're encouraging lazyness

**Comment by PJB3005** [comment]
No plastic.

File: code/datums/supplypacks.dm [hunk 1]
```diff
@@ -194,6 +194,41 @@ var/list/all_supply_groups = list("Supplies","Clothing","Security","Hospitality"
 	containername = "wooden planks crate"
 	group = "Supplies"
 
+/datum/supply_packs/plastic10
```
Comment (PJB3005) [inline code/datums/supplypacks.dm:197]: This type doesn't check out now does it?

File: code/modules/RCD/schematics/pipe.dm [hunk 2]
```diff
@@ -152,8 +152,8 @@
 
 	to_chat(user, "Building gas sensor...")
 	playsound(get_turf(master), 'sound/machines/click.ogg', 50, 1)
-	if(!do_after(user, A, 20))
-		return 1
+//	if(!do_after(user, A, 1))
```
Comment (PJB3005) [inline code/modules/RCD/schematics/pipe.dm:155]: You need to git gud.

**Comment by Probe1** [comment]
What did I teach you about making issues and discussing things before making balance changing PRs.

**Comment by Exxion** [comment]
There's no point when the PR doesn't take much effort to make. You can discuss in the PR.

**Comment by Kurfursten** [comment]
I do think it's fine to go ahead and just make the balance PR if it doesn't have a heavy time investment. That way if it gets thrown out, no big deal.

File: code/modules/RCD/schematics/pipe.dm (hunk 2 above)
Comment (jakmak) [reply code/modules/RCD/schematics/pipe.dm:155]: Wait what how the hell did the RPD file end up in this

Comment (PJB3005) [reply code/modules/RCD/schematics/pipe.dm:155]: I don't know I didn't make the PR.

Comment (jknpj) [reply code/modules/RCD/schematics/pipe.dm:155]: Maybe he made a branch of a branch

**Comment by Probe1** [comment]
If you want people to really dislike you yes, go ahead.  Be a dylan.

**Comment by Exxion** [comment]
That is completely not at all what Dylan's problem was

**Comment by SonixApache** [comment]
meteors

File: code/modules/RCD/schematics/pipe.dm (hunk 2 above)
Comment (jakmak) [reply code/modules/RCD/schematics/pipe.dm:155]: Does this mean I need to close the PR and re do it? Also I made the branch off of bleeding edge

Comment (PJB3005) [reply code/modules/RCD/schematics/pipe.dm:155]: No, you should be able to reset the branch and only recommit the changes you need.

**Comment by jakmak5** [comment]
OK I admit i made this PR literally as i was going out the door today so i didn't look at the files changed to make sure git didn't fuck up. I understand that this may have been not the best of ideas for a PR so i will close this for now

**Comment by Probe1** [comment]
@Exxion one of them definitely was "OK so I changed the balance because of X because i felt like it here's a PR"

"GOD WHY ARE PEOPLE SHOUTING MY PR DOWN"

**Comment by Exxion** [comment]
Yeah
The problem there is complaining about the discussion

**Comment by Probe1** [comment]
shrug

I stand by it.  If you don't want people to dislike you then you should open an issue before making a balance PR.  It lets people voice their opinion in a non committal way.  If you change the code and people dislike it then they're put up against the wall.  Either "I" go all out in trashing this PR or it gets merged and the game is worse in my opinion.  

Discussion can happen in a PR but for one that changes the balance, you really ought to have already discussed it with someone first.  

Obviously I don't see it as a rule and you don't have to adhere to it but my balance PRs rarely elicit any kind of controversy.

**Comment by jakmak5** [comment]
I assumed since it was such a minor thing that there wouldn't be as much of a outcry against it since plastic was an obscure resource used only by a single job which is easily acquired through scavenging parts of the station and the crates with 5 stacks of metal or glass was to reduce crate spam when mass buying materials.

**Pull request closed without merging**